Hephaestus/Utilities/ArgumentUtility.cs

```diff
@@ -47,103 +56,207 @@ internal static void Handle(string[] arguments)
             }
         }
 
+        /// <summary>
+        /// Displays the usage information.
+        /// </summary>
         private static void HelpCommand()
         {
-            Console.WriteLine("Available Hephaestus commands are:");
+            Console.WriteLine("Usage:");
             Console.WriteLine("    Launching with no commands will initiate building of your current project (if present).");
-            Console.WriteLine("    init    Initiate a new Hephaestus project at your current CD location.");
-            Console.WriteLine("    help    Display this again.");
+            Console.WriteLine("    init        Initiate a new Hephaestus project at your current CD location.");
+            Console.WriteLine("    force       Force build a project ignoring all checksums.");
+            Console.WriteLine("    version     Show Hephaestus version.");
+            Console.WriteLine("    help        Display this again.");
```

Comment:
Rather than multiple WriteLine, you can do it with one.

Can be done with either 
```C#
Console.WriteLine(@"Usage:
    Launching with no commands will initiate building of your current project (if present).
    init etcetcetc.");
```
or
```C#
Console.WriteLine(@"Usage:\n" +
"    Launching with no commands will initiate building of your current project (if present).\n" +
"    init etcetcetc.");
```

Reply:
I’d prefer the first option as it’s cleaner and doesn’t add all the string concatenation pluses.